### ISSUE TITLE
Rectangle shortcuts unavailable in VS Code

### ISSUE THREAD

**Issue by devnev** [rxhanson/Rectangle#167]
macOS version: Mojave, version 10.14.6
Rectangle version: Version 0.30 (35)
logs (In Rectangle menu, hold option, "View Logging..."): 
`2020-06-02T12:24:06+01:00: Unable to obtain accessibility element`

Since a few days ago, I can no longer use Rectangle to maximise my VS Code windows. Opening the rectangle menu while VS Code is focused shows all the actions greyed out. I assume this is due to a change in VS Code, but Rectangle seems like the right starting point for filing an issue as its Rectangle trying to do the resizing.

**Comment by rxhanson** [comment]
Did you try restarting vs code as noted in https://github.com/rxhanson/Rectangle/issues/43?

**Comment by devnev** [comment]
Thanks @rxhanson, that fixed it. I thought I'd looked for VS Code issues before filing, apparently I missed that one; sorry about that.
